docs/script.md

```diff
@@ -31,7 +31,7 @@ assert(script.toString() === '33 0x022df8750480ad5b26950b25c7ba79d3e37d75f640f8e
 ```
 
 ### Pay to Multisig (p2ms)
-Multisig outputs allow to share control of dashes between several keys. When creating the script, one specifies the public keys that control the funds, and how many of those keys are required to sign off spending transactions to be valid. An output with N public keys of which M are required is called an m-of-n output (For example, 2-of-3, 3-of-5, 4-of-4, etc.)
+Multisig outputs allow to share control of some funds between several keys. When creating the script, one specifies the public keys that control the funds, and how many of those keys are required to sign off spending transactions to be valid. An output with N public keys of which M are required is called an m-of-n output (For example, 2-of-3, 3-of-5, 4-of-4, etc.)
```

Comment:
I would remove "some" here.

Reply:
If you would, I will. It sounded better initially, but now it adds some conditionnal type of meaning, which is not what I wanted. 
What about `the funds` ? 
Control of funds is good ? It seems... wrong to ear, but I'm not a native speaker :x